docs/theming.md

```diff
@@ -89,53 +77,18 @@ Every template has access to a `log` function, allowing to log messages as usual
 
 ### CSS classes
 
-The *Material* theme uses the following CSS classes in the HTML:
-
-- `doc`: on all the following elements
-- `doc-children`: on `div`s containing the children of an object
-- `doc-object`: on `div`s containing an object
-    - `doc-attribute`: on `div`s containing an attribute
-    - `doc-class`: on `div`s containing a class
-    - `doc-function`: on `div`s containing a function
-    - `doc-method`: on `div`s containing a method
-    - `doc-module`: on `div`s containing a module
-- `doc-heading`: on objects headings
-- `doc-contents`: on `div`s wrapping the docstring then the children (if any)
-    - `first`: same, but only on the root object's contents `div`
-- `doc-properties`: on `span`s wrapping the object's properties
-    - `doc-property`: on `small` elements containing a property
-        - `doc-property-PROPERTY`: same, where `PROPERTY` is replaced by the actual property
-
-!!! example "Example with colorful properties"
-    === "CSS"
-        ```css
-        .doc-property { border-radius: 15px; padding: 0 5px; }
-        .doc-property-special { background-color: blue; color: white; }
-        .doc-property-private { background-color: red; color: white; }
-        .doc-property-property { background-color: green; color: white; }
-        .doc-property-read-only { background-color: yellow; color: black; }
-        ```
-
-    === "Result"
-        <style>
-          .prop { border-radius: 15px; padding: 0 5px; }
-        </style>
-        <h3 style="margin: 0;"><span>
-            <small class="prop" style="background-color: blue; color: white !important;">special</small>
-            <small class="prop" style="background-color: red; color: white !important;">private</small>
-            <small class="prop" style="background-color: green; color: white !important;">property</small>
-            <small class="prop" style="background-color: yellow; color: black !important;">read-only</small>
-        </span></h3>
-
-        As you can see, CSS is not my field of predilection...
+Since each handler provides its own set of templates, with their own CSS classes,
+we cannot list them all here. See the documentation about CSS classes for:
+
+- the Python handler: https://mkdocstrings.github.io/python/customization/#css-classes
```

Comment:
As stated, these were specific to the Python handler (moved there). Again, let me know if we can link to the Crystal handler's docs.

Reply:
Feel free to link to https://mkdocstrings.github.io/crystal/styling.html#custom-styles